src/pages/podcasts.js

```diff
@@ -1,11 +1,26 @@
+import PageLayout from '../components/PageLayout';
 import React from 'react';
-import cx from 'classnames';
 import SEO from '../components/Seo';
-import PageLayout from '../components/PageLayout';
+import cx from 'classnames';
 import podcastsHeader from '../images/podcasts/podcasts.jpg';
 import styles from './podcasts.module.scss';
 
 const PodcastsPage = () => {
+  const podcastIds = ['1225223', '1677727', '1677670'];
+
+  const podcastEmbeds = podcastIds.map((podcastId) => {
+    return (
+      <section className={cx(styles.section, styles.player)} key={podcastId}>
+        <div>
```

Comment:
[nit] It looks like this used to have an `id` (which looked useless, so thanks so much for removing that!). Now that this is empty, do we need this wrapping div? Could we instead just put the `iframe` here?